sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -637,35 +637,120 @@ SIGVERIFY_MOD_EXP_CASES = [
     {
         "name": "sw",
         "use_sw_rsa_verify": CONST.TRUE,
-        "exit_success": MSG_PASS,
+        "exit_success": {lc_state.lower(): "use_sw_rsa_verify=0x00000739" for lc_state in structs.to_dict(CONST.LCV)},
+    },
+    {
+        "name": "otbn",
+        "use_sw_rsa_verify": CONST.FALSE,
+        "exit_success": {lc_state.lower(): "use_sw_rsa_verify=0x000001d4" for lc_state in structs.to_dict(CONST.LCV)},
+    },
+    {
+        "name": "invalid",
+        "use_sw_rsa_verify": 0,
+        "exit_success": {
+            lc_state.lower(): MSG_TEMPLATE_BFV.format(hex(CONST.BFV.INTERRUPT.ILLEGAL_INSTRUCTION)) if lc_state_val != CONST.LCV.TEST_UNLOCKED0 else MSG_PASS
+            for lc_state, lc_state_val in structs.to_dict(CONST.LCV).items()
+        },
     },
-    # TODO(#14481): Enable after OTP splicing is ready.
-    # TODO: Consider using a custom binary that prints the OTP item.
-    # {
-    #     "name": "otbn",
-    #     "use_sw_rsa_verify": CONST.FALSE,
-    #     "exit_success": MSG_PASS,
-    # },
-    # {
-    #     "name": "invalid",
-    #     "use_sw_rsa_verify": 0,
-    #     "exit_success": MSG_TEMPLATE_BFV.format(CONST.BFV.INTERRUPT.ILLEGAL_INSTRUCTION),
-    # },
 ]
 
-[opentitan_functest(
-    name = "sigverify_mod_exp_{}".format(t["name"]),
-    cw310 = cw310_params(
-        exit_success = t["exit_success"],
-    ),
-    ot_flash_binary = ":empty_test_slot_a",
-    targets = ["cw310_rom"],
-) for t in SIGVERIFY_MOD_EXP_CASES]
+opentitan_flash_binary(
+    name = "empty_test_sigverify_mod_exp",
+    srcs = ["empty_test.c"],
+    devices = ["fpga_cw310"],
+    local_defines = [
+        shell.quote("EMPTY_TEST_MSG=\"use_sw_rsa_verify=0x%08x\", otp_read32(OTP_CTRL_PARAM_CREATOR_SW_CFG_USE_SW_RSA_VERIFY_OFFSET)"),
+    ],
+    signed = True,
+    deps = [
+        "//hw/ip/otp_ctrl/data:otp_ctrl_regs",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+        "//sw/device/silicon_creator/lib/drivers:otp",
+        "@//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
```

Comment:
Is the `@` prefix necessary?

Reply:
I'm not sure, it was there when I lifted these from `opentitan_test.bzl` and thought they could be relevant for manufacturing hooks. But I'm not sure if we need to keep it here. @timothytrippel any thoughts? (apologies if this is the n^{th} time I'm asking about this)

Reply:
the `@` were only necessary in `opentitan_test.bzl` because an external Bazel repository was using certain macros / targets (the manufacturer hooks repo). Since there are no external repositories relying on things defined in this file you shouldn't need them here (applies throughout the file). The `@//` is short for `@lowrisc_opentitan//`.

Reply:
Thanks for the clarification @timothytrippel!